types/serverless/index.d.ts

```diff
@@ -0,0 +1,42 @@
+// Type definitions for serverless 1.18
+// Project: https://github.com/serverless/serverless#readme
+// Definitions by: Sebastian Müller <https://github.com/sbstjn>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace Serverless {
+    interface Options {
+        stage: string | null;
+        region: string | null;
+        noDeploy?: boolean;
+    }
+
+    namespace Provider {
+        class Aws {
+            constructor(serverless: Serverless, options: Serverless.Options)
+
+            getProviderName: () => string;
+            getRegion: () => string;
+            getServerlessDeploymentBucketName: () => string;
+            getStage: () => string;
+        }
+    }
+}
+
+interface Serverless {
+    cli: {
+        log(message: string): null
+    };
+
+    config: {
```

Comment:
And type here probably needs to be of type class Config since this is something that matches  https://github.com/serverless/serverless/blob/master/lib/classes/Config.js